api/src/main/java/io/strimzi/api/kafka/model/MaintenanceSpec.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MaintenanceSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private List<String> timeWindows;
+
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("A list of time windows for the maintenance tasks. Each time window is defined by a cron expression.")
```

Comment:
We definitely need to define or refer to other documentation of what the supported syntax is.

Reply:
I didn't find a good link that wasn't a blog post or something like that. Have you a good link for this? Happy to use it :-)